digitalidentity/service.go

```diff
@@ -256,17 +251,9 @@ func GetShareReceipt(httpClient requests.HttpClient, receiptId string, clientSdk
 		return receipt, fmt.Errorf("failed to build application extra data: %v", err)
 	}
 
-	uattr, err := cryptoutil.DecryptReceiptContent(receiptResponse.OtherPartyContent.Profile, receiptContentKey)
-	if err != nil {
-		return receipt, fmt.Errorf("failed to decrypt other party receipt content profile: %v", err)
-	}
-	uextra, err := cryptoutil.DecryptReceiptContent(receiptResponse.OtherPartyContent.ExtraData, receiptContentKey)
+	uattrData, uextra, err := decryptReceiptContent(receiptResponse.OtherPartyContent, receiptContentKey)
 	if err != nil {
-		return receipt, fmt.Errorf("failed to decrypt other party receipt content extra data: %v", err)
-	}
-	uattrData := &yotiprotoattr.AttributeList{}
-	if err := proto.Unmarshal(uattr, uattrData); err != nil {
-		return receipt, fmt.Errorf("failed to unmarshal attribute list: %v", err)
+		return receipt, err
```

Comment:
make sure to add context to both of these errors (here and above)

e.g. 
```golang
return receipt, fmt.Errorf("failed to decrypt other party receipt content :: %v", err)
```

Reply:
updated